### ISSUE TITLE
javascript strings not being translated

### ISSUE THREAD

**Issue by robert81** [Melapress/admin-notices-manager#117]
Ticket: https://wordpress.org/support/topic/issue-with-javascript-strings-not-being-translated/

**Comment by DannyWPWS** [comment]
Yeah spotted quite a few when working on recent fixes, will sort now

**Comment by jwjsystem** [comment]
checked the code and looks fixed...https://github.com/Melapress/admin-notices-manager/pull/125
